src/components/AddressSearch.js

```diff
@@ -37,6 +37,7 @@ const defaultProps = {
 
 const AddressSearch = (props) => {
     const googlePlacesRef = useRef();
+    const [display, setDisplay] = useState(false);
```

Comment:
NAB: Consider improving the name of the state

Reply:
Updated to `displayListViewBorder`